### DISCUSSION TITLE
Added Config in UI

### DISCUSSION THREAD

**Discussion by Ethan-Otto** [assafelovic/gpt-researcher#251] [Show and tell]
![image](https://github.com/assafelovic/gpt-researcher/assets/89815515/08c87bad-3990-49b3-933e-787b3f31161c)
![image](https://github.com/assafelovic/gpt-researcher/assets/89815515/0acf4e57-e552-4dbd-b6a7-8b0e36a243c3)

Added UI for controlling the config. I made the config.json in the root folder the default config, and the UI can update it directly. Allows much faster testing different parameters especially when using docker. 
Also was considering adding descriptions, note some config parameters appear unimplemented. Let me know if I am off with these.  I'd be happy to make a pull request if people like it.  

type_description  ={
    "debug_mode": " (NOT IMPLEMENTED)",
    "allow_downloads": "(NOT IMPLEMENTED)",
    "selenium_web_browser": "",
    "retriever": "",
    "llm_provider": "",
    "fast_llm_model": "",
    "smart_llm_model": "",
    "fast_token_limit": "(NOT IMPLEMENTED)",
    "smart_token_limit": "(NOT IMPLEMENTED)",
    "browse_chunk_max_length": "",
    "summary_token_limit": "",
    "temperature": "creativity measure",
    "user_agent": "",
    "memory_backend": "",
    "total_words": "Minimum Total words in Report",
    "report_format": "",
    "max_iterations": "Number of phrases generated then searched"
}

**Comment by assafelovic** [comment]
This is cool @Ethan-Otto ! How about opening a PR?

**Reply by Ethan-Otto** [reply]
Awesome, Just opened!
